src/index.html

```diff
@@ -7,8 +7,24 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home"><img src="images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="is-active">apple</a></li>
+          <li class="nav__item"><a href="#samsung">samsung</a></li>
```

Comment:
Need to add class `nav__link` to tag a, and use a selector with the classes in the styles
```suggestion
          <li class="nav__item">
            <a href="#samsung" class="nav__link">samsung</a>
          </li>
```